### ISSUE TITLE
Don't understand how mask works

### ISSUE THREAD

**Issue by richb-hanover** [timwaters/mapwarper#184]
I have created a map and imported it into Gaia GPS. It works wonderfully - the warping process puts things exactly where I thought they would be. It's at: http://mapwarper.net/maps/46805

But... The map has a white border that obscures features that aren't part of the map. So I figured that the Map Mask option would allow me to retain only the interesting part of the map, and "cut away" the uninteresting border. 

I used the Mask option, and re-saved the map after re-rectifying it. But the image that appears in GaiaGPS still retains the white border. Am I doing something wrong? Thanks.

**Comment by DonpK** [comment]
I agree on both counts. MapWarper works beautifully. I'm very happy with the results on some 200 year old maps where it's clear the original survey was anything but precise. 

I have had some problems using the Mask function.  Sometimes it works fine, in other cases the map becomes unusable. Clearly I'm doing something wrong, but I'm not sure what.  So perhaps some do's and don'ts on the Mask page would be helpful.

**Comment by richb-hanover** [comment]
It worked when I uploaded a second map ( http://mapwarper.net/maps/46850 ), created a mask, and rectified it.

So I went back to the original map, followed the same procedure (I think) but it did not get a mask... I would be intrigued to hear what the difference between the two maps are. Thanks.

**Comment by fox91** [comment]
I have the same issue with my local installation.
When I save the mask in the log I see this error:
```
ERROR gdal rasterize script: ERROR 1: Unable to compute a transformation between pixel/line
and georeferenced coordinates for /srv/mapwarper/public/mapimages/src/mapname.jpg.tif_masked.
There is no affine transformation and no GCPs.
```
I don't know if this error is related with this issue.
